slack/resource_usergroup_channels.go

```diff
@@ -40,7 +40,7 @@ func resourceSlackUserGroupChannels() *schema.Resource {
 
 func configureSlackUserGroupChannels(d *schema.ResourceData, userGroup slack.UserGroup) {
 	d.SetId(userGroup.ID)
-	_ = d.Set("channels", userGroup.Prefs.Channels)
+	_ = d.Set("channels", append(userGroup.Prefs.Channels, userGroup.Prefs.Groups...))
```

Comment:
I've confirmed https://api.slack.com/methods/usergroups.create and https://api.slack.com/methods/usergroups.update take just `channels` and they aren't distinguished.